### PR TITLE
Fix loading bugs [urgent]

### PR DESCRIPTION
**Pull request by Raemon**
I accidentally put the loading-spinner in a place it couldn't show up. Also, the voteTotal wasn't loading properly if you navigated to the page from the frontpage.

### CODE REVIEW

**Comment by Discordius** [comment]
The lint errors seem real

**Comment by Raemon** [comment]
Fixed, although I'm still a bit confused about how to relate to useEffect dependency arrays (it's not obvious to me I want this to fire more than once, although I guess maybe that's appropriate)

**Comment by Raemon** [comment]
merging since the bug is live.

File: packages/lesswrong/components/review/ReviewVotingPage.tsx [hunk 1]
```diff
@@ -546,11 +551,11 @@ const ReviewVotingPage = ({classes}: {
                   </span>
                 </LWTooltip> 
                 {getReviewPhase() !== "VOTING" && <>({sortedPosts.length} Nominated)</>}
-                {(postsLoading || loading) && <Loading/>}
               </div>
             }
+            {(postsLoading || loading) && <Loading/>}
```
Comment (jpaddison3) [inline packages/lesswrong/components/review/ReviewVotingPage.tsx:556]: This loading spinner is not positioned correctly on either LW or the EA Forum

**Review by jpaddison3 — COMMENTED** [review]
[Post-Merge Review] Found an issue

**Pull request merged**